src/test/java/uk/nhs/tis/trainee/notifications/service/ProgrammeMembershipServiceTest.java

```diff
@@ -688,6 +688,41 @@ void shouldAddDirectNotificationsWhenNotExcluded() throws SchedulerException {
     assertThat("Unexpected start date.", jobDataMap.get(START_DATE_FIELD), is(START_DATE));
     assertThat("Unexpected CoJ synced at.", jobDataMap.get(COJ_SYNCED_FIELD),
         is(Instant.MIN));
+
+    // PROGRAMME_DAY_ONE
+    ArgumentCaptor<String> dayOneStringCaptor = ArgumentCaptor.captor();
+    ArgumentCaptor<JobDataMap> dayOneJobDataMapCaptor = ArgumentCaptor.captor();
+    verify(notificationService).scheduleNotification(
+        dayOneStringCaptor.capture(),
+        dayOneJobDataMapCaptor.capture(),
+        eq(Date.from(START_DATE.atStartOfDay(timezone).toInstant())));
+
+    String dayOneJobId = dayOneStringCaptor.getValue();
+    String dayOneExpectedJobId = PROGRAMME_DAY_ONE + "-" + TIS_ID;
+    assertThat("Unexpected job id.", dayOneJobId, is(dayOneExpectedJobId));
+
+    JobDataMap dayOneJobDataMap = dayOneJobDataMapCaptor.getValue();
+    assertThat("Unexpected tisId.", dayOneJobDataMap.get(TIS_ID_FIELD), is(TIS_ID));
```

Comment:
Check the map size here to ensure we don't include any additional fields that aren't expected.